backend/open_webui/utils/middleware.py

```
@@ -967,22 +1027,84 @@ async def background_tasks_handler():
+                            )
+
+                            await event_emitter(
+                                {
+                                    "type": "chat:message:follow_ups",
+                                    "data": {
+                                        "follow_ups": follow_ups,
+                                    },
+                                }
+                            )
+                        except Exception as e:
+                            pass
```

Ruff (0.11.9)

`try`-`except`-`pass` detected, consider logging the exception

(S110)